src/cljs/org/broadinstitute/firecloud_ui/paths.cljs

```diff
@@ -45,5 +45,8 @@
 (defn submission-details [workspace-id submission-id]
   (str (submissions-list workspace-id) "/" submission-id))
 
+(defn submission-kill [workspace-id submission-id]
```

Comment:
Yes, it probably is redundant. That redundancy should be addressed as a separate refactoring task though - not in individual commits.